tests/functionnal/DbUtils.php

```diff
@@ -42,6 +42,15 @@ class DbUtils extends DbTestCase {
       'testGetSonsOfCached'
    ];
 
+   public function beforeTestMethod($method) {
+      parent::beforeTestMethod($method);
+      if (in_array($method, $this->cached_methods)) {
+         $_SESSION['glpicronuserrunning'] = "cron_phpunit";
+      } else {
+         $this->login();
+      }
+   }
```

Comment:
Are all methods requiring a login ?

Reply:
Hum... I'll test again, I made that change at the very beginning of the work

Reply:
Ok, changed